ImportError: cannot import name 'Any' from 'soma.controller.field'

New capsul CI error, for example in job [22350418504](https://github.com/populse/capsul/actions/runs/8174796064/job/22350418504?pr=359) of https://github.com/populse/capsul/pull/359:
```console
ImportError while importing test module '/home/runner/work/capsul/capsul/capsul/pipeline/test/test_complex_pipeline_activations.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
pipeline/test/test_complex_pipeline_activations.py:7: in <module>
    from soma.controller import File, field
/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/soma/controller/__init__.py:15: in <module>
    from .field import (
E   ImportError: cannot import name 'Any' from 'soma.controller.field' (/opt/hostedtoolcache/Python/3.9.18/x64/lib/python3.9/site-packages/soma/controller/field.py)
```
This looks related to my recent changes in soma-base, but the truth is I cannot find `Any` in [soma/controller/field.py](https://github.com/populse/soma-base/blob/6.0/python/soma/controller/field.py), either the last version or prior versions.

--- a/python/soma/controller/field.py
+++ b/python/soma/controller/field.py
@@ -5,6 +5,8 @@ import typing
 
 # Import allsupported types from typing
 from typing import (
+    Any,
+    Literal,
     Union,
 )
 

--- a/python/soma/proxy.py
+++ b/python/soma/proxy.py
@@ -40,9 +40,9 @@ class ContainerWithProxy:
         super().__init__()
         self.proxy_values = proxy_values
         self.content = content
-        if all_proxies == True:
+        if all_proxies is True:
             all_proxies = {}
-        elif all_proxies == False:
+        elif all_proxies is False:
             all_proxies = None
         self.all_proxies = all_proxies
 

--- a/python/soma/proxy.py
+++ b/python/soma/proxy.py
@@ -40,7 +40,11 @@ class ContainerWithProxy:
         super().__init__()
         self.proxy_values = proxy_values
         self.content = content
-        self.all_proxies = {} if all_proxies else None
+        if all_proxies == True:
+            all_proxies = {}
+        elif all_proxies == False:
+            all_proxies = None
+        self.all_proxies = all_proxies
 
     def proxy(self, value):
         """

